service.subtitles.subztv.gr/resources/lib/subztvgr.py

```diff
@@ -0,0 +1,168 @@
+
+                url = client.parseDOM(result, 'div', attrs = {'id': 'series'})[0]
+                url = re.findall('(/series/\d+)', url)
+                url = [x for y,x in enumerate(url) if x not in url[:y]]
+                url = [urlparse.urljoin('http://subztv.gr', i) for i in url]
+                url = url[:3]
+
+                for i in url:
+                    try:
+                        t, y = cache.get(self.cache, 2200, i)
+
+                        if cleantitle.get(t) == cleantitle.get(title):
+                            item = i ; break
+                    except:
+                        pass
+
+                item = '%s/seasons/%s/episodes/%s' % (item, season, episode)
+                item = client.request(item)
+
+
+            item = re.sub(r'[^\x00-\x7F]+', ' ', item)
+            items = client.parseDOM(item, 'tr', attrs = {'data-id': '\d+'})
+        except:
+            return
+
+        for item in items:
+            try:
+                if not 'img/el.png' in item: raise Exception()
+
+                name = client.parseDOM(item, 'td', attrs = {'class': '.+?'})[-1]
+                name = name.split('>')[-1].strip()
+                name = re.sub('\s\s+', ' ', name)
+                name = client.replaceHTMLCodes(name)
+                name = name.encode('utf-8')
+
+                url = re.findall('\'(http(?:s|)\://.+?)\'', item)[-1]
+                url = client.replaceHTMLCodes(url)
+                url = url.encode('utf-8')
+
+                self.list.append({'name': name, 'url': url, 'source': 'subztvgr', 'rating': 5})
+            except:
+                pass
+
+        return self.list
+
+
+    def cache(self, i):
+        try:
+            self.r = client.request(i)
+            self.r = re.sub(r'[^\x00-\x7F]+', ' ', self.r)
+            t = re.findall('(?:\"|\')original_title(?:\"|\')\s*:\s*(?:\"|\')(.+?)(?:\"|\')', self.r)[0]
+            y = re.findall('(?:\"|\')year(?:\"|\')\s*:\s*(?:\"|\'|)(\d{4})', self.r)[0]
+            return (t, y)
+        except:
+            pass
+
+
+    def download(self, path, url):
+        try:
+            result = client.request(url)
+
+            f = os.path.splitext(urlparse.urlparse(url).path)[1][1:]
+            f = os.path.join(path, 'file.%s' % f)
+
+            file = open(f, 'wb') ; file.write(result) ; file.close()
+
+            dirs, files = control.listDir(path)
+
+            if len(files) == 0: return
+
+            control.execute('Extract("%s","%s")' % (f, path))
+
+            for i in range(0, 10):
+                try:
+                    dirs, files = control.listDir(path)
+                    if len(files) > 1: break
+                    time.sleep(1)
+                except:
+                    pass
+
+            control.deleteFile(f)
+
+            subtitle = [i for i in files if any(i.endswith(x) for x in ['.srt', '.sub'])][0]
+
+            try: subtitle = os.path.join(path, subtitle)
```

Comment:
There is a greek subtitle team that uses on it's naming acute accents (polytonic greek)
https://en.wikipedia.org/wiki/Greek_diacritics
I tried with decode('utf-8'), but python can't handle it since it's not a part of modern greek
The only easy way i found for python to handle files with these accents and save them was this
utf-8 failed all the time